iosApp/iosApp/Util.swift

```diff
@@ -2,7 +2,7 @@ import SwiftUI
 import Shared
 import Foundation
 
-func formatTime(localDateTime: Date) -> String {
+func dateTimeFormatter(localDateTime: Date) -> String {
```

Comment:
I think you misunderstood the comment, the function name was clear but the file naming was a bit too generic, all these functions in the Util file are related to date and time. It should be a bit more seperated with a DateTimeFormatter file